docs/modules/migrate/pages/dmt-migration.adoc

```diff
@@ -1,8 +1,10 @@
 = Migrate Your Data
-:description: The DMT migrates your data for maps and replicated maps only, and can be run on Mac, Linux, and Windows Operating Systems.
+:description: The DMT migrates your data for maps and replicated maps, compact serialization schemas, and- if configured - CP data structures. It can be run on Mac, Linux, and Windows Operating Systems.
```

Comment:
for now, the cp configuration is only txt files where you list their names. we can try to allow them not requiring a cluster restart if they give us a list of names instead of a wildcard. then there will be a config.json option for that. 

but as I know, we'll not do it. am i right? @frant-hartm

Reply:
this means for now, we can delete "if configured" part above. if there wont be a extra config for cp structures in config.json